scripts/generate/test_generate.py

```diff
@@ -127,14 +126,12 @@ def test_generate(
             pp=1,
             ep=1,
             world_size=world_size,
-            enable_loss_parallel=False,
         )
-        # Build world mesh for parallelism
-        world_mesh = parallel_dims.build_mesh(device_type=device_type)
+        world_mesh = parallel_dims.world_mesh
```

Comment:
nit: We don't need this line here to "build world mesh" explicitly, right?  In line 134, `parallel_dims.world_mesh["tp"]` will call `build_mesh` internally.